Classes/Presentation/Modules/SearchListItemsFactory.swift

```diff
@@ -24,9 +25,13 @@ final class SearchListItemsFactory {
             guard let translation = meaning.translation?.text else {
                 return nil
             }
-            return SearchCollectionViewCellItem(translation: translation)
+            let cellItem = SearchCollectionViewCellItem(translation: translation)
+            cellItem.diffIdentifier = UUID().uuidString
+            cellItem.itemDidSelectHandler = { [weak output] _ in
+                output?.openTranslationViewController()
```

Comment:
Я бы переименовал эту функцию, тут обрабатывается нажатие на ячейку, потом только открытие модуля
```suggestion
                output?.selectCell()
```